static/locales/ja.json

```
@@ -1631,13 +1631,15 @@
         "exportSettingsFailed": "設定のエクスポートに失敗しました: {{error}}",
         "capturingPortrait": "立ち絵をキャプチャ中...",
         "generatingCard": "キャラクターカードを生成中...",
+        "idleMotionHint": "保存時、選択したモーションが待機モーションとして設定されます",
```

Biome (2.4.13)

[error] The key idleMotionHint was already declared.

(lint/suspicious/noDuplicateObjectKeys)